### ISSUE TITLE
Down load in windows system 

### ISSUE THREAD

**Issue by clun1** [WISDEM/CCBlade#27]
May I ask whether ccblade can be downloaded on Windows system? If yes, what should I pay attention to

**Comment by gbarter** [comment]
Yes, CCBlade can be built on Windows.  If you have already done `conda install wisdem` then you have already installed CCBlade.

**Comment by clun1** [comment]
叶片，i download wisdem,and run the code like this,but i don't kown why it failed?
<img width="853" alt="image" src="https://user-images.githubusercontent.com/93849049/231376410-a5d807af-b8c1-442f-bba0-48f7a48c67ca.png">

**Comment by clun1** [comment]
yep，i download wisdem,and run the ccblade like this,but i don't know why it failed?
<img width="853" alt="image" src="https://user-images.githubusercontent.com/93849049/231376639-da68afe2-df8f-4da8-ace1-eabd108f288b.png">

**Comment by clun1** [comment]
i'm curious if downloading wisdem or ccblade requires a separate download of fortran complier?

**Comment by gbarter** [comment]
On your error, python is complaining that it cannot find the file you have specified.  That is not a WISDEM or CCBlade error.  Double check your file spelling and paths.

If you want to build WISDEM and CCBlade from the source, you do need a Fortran compiler (which is available via Anaconda).  If you did `conda install wisdem`, then you don't need a separate command to install a Fortran compiler.

**Comment by clun1** [comment]
thank for your guidance,but i have a new problem,and i don't know why it happen
<img width="746" alt="image" src="https://user-images.githubusercontent.com/93849049/231499988-7d09ef79-f4ac-4334-a565-6fa430cadd06.png">
<img width="577" alt="image" src="https://user-images.githubusercontent.com/93849049/231500109-22636499-53fa-4130-9526-828bdcab2cb5.png">

**Comment by gbarter** [comment]
If you installed CCBlade via WISDEM, then you have to access CCBlade via the `wisdem` python package and scripts.  Try this one instead:
https://github.com/WISDEM/WISDEM/tree/master/wisdem/test/test_ccblade

**Comment by clun1** [comment]
thank for your guidance,i can run this example,but there are some alarms, i do not kown whether there will be any impact and how to solve these alarms
<img width="905" alt="image" src="https://user-images.githubusercontent.com/93849049/231727964-4c446502-4a15-4b1f-9f1c-ebb6a421a401.png">
<img width="919" alt="image" src="https://user-images.githubusercontent.com/93849049/231728012-6d7f97c2-858d-4c4f-9581-fb0e1c7089f9.png">

**Comment by gbarter** [comment]
There is no requirement for `mpi4py` and your WISDEM installation was working fine in that message.  I'm not sure what changed to your most recent message.  Be sure you are working in the correct Anaconda environment where you installed WISDEM (`conda activate name-of-your-wisdem-environment`).
